ecosystem/platform/server/app/controllers/health_controller.rb

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+# frozen_string_literal: true
+
+class HealthController < ApplicationController
+  def health
+    User.where(id: 1).exists?
+    render plain: '🔥🤼🔥'
```

Comment:
??? I don't understand this

Reply:
health endpoint; verifies DB connect works and server is serving. Will be called by AWS; if it fails, container gets relaunched.

If you're wondering why it's two people wrestling surrounded by flames, I figured page needs to return something, lets amuse anyone who tries calling `/health` on random servers to see what happens :-P